### DISCUSSION TITLE
邮箱通知问题

### DISCUSSION THREAD

**Discussion by HaoqinChen** [walinejs/waline#1079] [Q&A] [answered]
你好，我进行邮箱配置后评论没有通知，请问可能是什么原因呢？
![image](https://user-images.githubusercontent.com/90776957/171529709-5b7984b3-27a2-47e7-b1d2-984cc79fe1c7.png)
密码是163邮箱开启pop/stmp后给予的授权码
我暂时用微信通知进行代替。

**Comment by HaoqinChen** [comment]
![image](https://user-images.githubusercontent.com/90776957/171534242-31f2ff7f-a778-42d9-b000-66cdd33e1d0f.png)
这是我的全部环境变量，我评论他人可以实现发邮件提醒他人，但他人评论我收不到提醒
是因为我开了评论审核吗？

**Comment by lizheming** [comment, marked as answer]
1. 博主通知，非 AUTHOR_EMAIL 发布的评论都会给 AUTHOR_EMAIL 的评论发布通知
2. 评论回复通知，审核中的评论是不发的，审核通过之后会发通知
